automerge-backend/src/lib.rs

```diff
@@ -32,17 +32,12 @@ pub use error::AutomergeError;
 
 #[cfg(test)]
 mod tests {
-    use std::{
-        sync::{Arc, Mutex},
-        thread,
-    };
+    use std::thread;
 
     #[test]
-    fn sync_and_send_backend() {
+    fn test_backend_is_send() {
```

Comment:
It appears to me that it's sufficient to test that the backend is `Send`, since by definition anything that is can be made `Sync` by putting a lock around it. So the previous test doesn't actually test that the backend is `Sync`, it makes it so by putting the mutex around it. 

If one wanted to test for a `Sync` back-end, one could do it by putting only an `Arc` around it.